posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc

```diff
@@ -0,0 +1,118 @@
+= "Hibernate OGM 5.3.1.Final is out"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We're happy to announce the release of:
+http://hibernate.org/ogm/releases/5.3/#get-it[Hibernate OGM 5.3.1.Final]
+
+== What's new compared to 5.3.0.Final?
+
+This ia a fix release for native queries pagination and projection issues.
```

Comment:
s/ia/is/
s/fix/maintenance/
something like:
`This is a maintenance release fixing issues related to native queries pagination and projection.`